routing/routing_session.cpp

```diff
@@ -88,10 +88,17 @@ void RoutingSession::RebuildRoute(m2::PointD const & startPoint,
   CHECK(m_router, ());
   RemoveRoute();
   SetState(routeRebuildingState);
-  m_routingRebuildCount++;
-  m_lastCompletionPercent = 0;
 
-  m_checkpoints.SetPointFrom(startPoint);
+  {
+    // TODO: after moving all routing callbacks to single thread this guard can be removed
```

Comment:
// @TODO(bykoianko) After moving all routing callbacks to single thread this guard can be removed.